v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/KafkaToGCSOptions.java

```diff
@@ -0,0 +1,49 @@
+package com.google.cloud.teleport.v2.transforms;
```

Comment:
This should be in the kafka-to-gcs module and not in common. Also this is an option and not a transform so should be in a separate options package.